converters/toZigbee.js

```diff
@@ -1242,6 +1242,26 @@ const converters = {
             entity.commandResponse('ssIasAce', 'panelStatusChanged', payload);
         },
     },
+    // set the status of keypad in accordance with the assigned Zigbee standard IAS status'
+    set_status: {
+        key: ['set_status'],
+        convertSet: async (entity, key, value, meta) => {
+            const panelStatus = utils.getKeyByValue(common.panStat, value.panelstatus, undefined);
```

Comment:
Please make everything in value snake case, e.g. `panelStatus` -> `panel_status`.